### PR TITLE
integrated all latest feature of graph control

### PR DESCRIPTION
**Pull request by sagarbag6132401**
sampleApp1
sampleApp2

### CODE REVIEW

File: src/components/tf-graph-datasource/tf-graph-datasource.tsx [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+                            color: `${traceColor}`,
+                            opacity: markerThickness
+                        },
+                        yAxisIndex: yAxisIndex ? yAxisIndex : null,                    
+                    }
+                    updatedTraceData.push(singleTraceData);
+                }
+            }
+        }
+        console.log("🚀 ~ file: tf-graph-datasource.tsx:461 ~ TfGraphDatasource ~ getTraceData ~ updatedTraceData", updatedTraceData)
```
Comment (pronayguha13) [inline src/components/tf-graph-datasource/tf-graph-datasource.tsx]: @sagarbag6132401  please remove the console log

File: src/components/tf-graph-datasource/tf-graph-datasource.tsx [hunk 2]
```diff
@@ -0,0 +1,529 @@
+import { Component, h, Prop } from '@stencil/core';
```
Comment (pronayguha13) [inline src/components/tf-graph-datasource/tf-graph-datasource.tsx:2]: @sagarbag6132401  please organise the imports

File: src/js/graphControl.js [hunk 3]
```diff
@@ -1,163 +1,332 @@
...
-const graphControl = document.querySelector('#GraphControl');
-graphControl.dataZoom = [
-  {
-    type: 'inside'
-  }
-]
-graphControl.tooltip = {
-  axisPointer: {
-    type: "cross"
+let traceType = ['line', 'scatter', 'bar', 'lineWithErrorBar', 'scatterWithErrorBar'];
```
Comment (pronayguha13) [inline src/js/graphControl.js]: @sagarbag6132401  array is a collection, the name of identifier cannot be in singular

File: src/js/graphControl.js [hunk 4]
```diff
@@ -1,163 +1,332 @@
...
+/*---------------------sample graph for graph control start--------------------*/
+const dataPoints = [[5,7],[7,8],[15,10],[20,18],[17,21]]; 
+const getErrorData = () =>{
```
Comment (pronayguha13) [inline src/js/graphControl.js]: @sagarbag6132401 please provide documentation

File: src/js/graphControl.js [hunk 5]
```diff
@@ -1,163 +1,332 @@
...
-graphControl.yAxis = {
-  type: 'value',
-  name: "Price",
-  nameLocation: "middle",
-  nameTextStyle: {
-    fontSize: "10",
-    color: "black",
-    align: "center",
-    lineHeight: "56",
+const GraphControlDemo = document.querySelector('#GraphControlDemo');
```
Comment (pronayguha13) [inline src/js/graphControl.js:58]: @sagarbag6132401  couple of issues with the naming convention here 

- Make it camelCase
- It is not a very proper naming convention

File: src/js/graphControl.js [hunk 6]
```diff
@@ -1,163 +1,332 @@
...
-  nameTextStyle: {
-    fontSize: "10",
-    color: "black",
-    align: "center",
-    verticalAlign: "top",
-    lineHeight: "56"
+const traceData = {
+    traceID: "0",
+    traceColor: "rgb(8,138,6)",
+    traceType: "line",
```
Comment (pronayguha13) [inline src/js/graphControl.js:36]: @sagarbag6132401 please use enums

**Review by pronayguha13 — DISMISSED** [review]

File: src/js/graphControl.js (hunk 4 above)
Comment (sagarbag6132401) [reply src/js/graphControl.js]: Updated the documentation

File: src/js/graphControl.js (hunk 3 above)
Comment (sagarbag6132401) [reply src/js/graphControl.js]: traceType array is not used anymore.

File: src/components/tf-graph-datasource/tf-graph-datasource.tsx (hunk 1 above)
Comment (sagarbag6132401) [reply src/components/tf-graph-datasource/tf-graph-datasource.tsx]: I have removed all the console log.

File: src/components/tf-graph-datasource/tf-graph-datasource.tsx (hunk 2 above)
Comment (sagarbag6132401) [reply src/components/tf-graph-datasource/tf-graph-datasource.tsx:2]: I have organized the imports.

File: src/js/graphControl.js (hunk 5 above)
Comment (sagarbag6132401) [reply src/js/graphControl.js:58]: Will take it up later.

File: src/js/graphControl.js (hunk 6 above)
Comment (sagarbag6132401) [reply src/js/graphControl.js:36]: Will take it up later.

**Review by pronayguha13 — APPROVED** [review]

**Pull request merged**